File: SAiCSInnovationsAPI 3.0 -test/ViewModels/NewCourseVM.cs
using SAiCSInnovationsAPI_3._0.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SAiCSInnovationsAPI_3._0.ViewModels
{
    public class NewCourseVM
    {
        public Course Course { get; set; }
        public List<SectionContent> SectionContent { get; set; }
        public Quiz Quiz { get; set; }
        public List<QuestionBank> QuestionBank { get; set; }
    }
}

File: MobileApp/src/app/Models/ViewModels/NewCourseVM.ts
import { SectionContent } from "../SectionContent";
import { Course } from "../Course";
import { Option } from "../Option";
import { QuestionBank } from "../QuestionBank";
import { Quiz } from "../Quiz";
import { Section } from "../Section";

export class NewCourseVM{
        Course:Course
        SectionContent:SectionContent[]
        Quiz:Quiz
        QuestionBank:QuestionBank[]
}